src/test/java/org/prebid/server/bidder/infytv/InfytvBidderTest.java

```diff
@@ -0,0 +1,159 @@
+package org.prebid.server.bidder.infytv;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import io.netty.handler.codec.http.HttpHeaderValues;
+import io.vertx.core.http.HttpMethod;
+import org.junit.Before;
+import org.junit.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.UnaryOperator;
+
+import static java.util.Collections.singletonList;
+import static java.util.function.UnaryOperator.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.api.Assertions.tuple;
+
+public class InfytvBidderTest extends VertxTest {
+
+    private static final String ENDPOINT_URL = "https://nxs.infy.tv/pbs/openrtb";
+
+    private InfytvBidder infytvBidder;
+
+    @Before
+    public void setUp() {
+        infytvBidder = new InfytvBidder(ENDPOINT_URL, jacksonMapper);
+    }
+
+    @Test
+    public void makeHttpRequestsShouldCreateExpectedUrl() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(identity());
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = infytvBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getUri)
+                .containsExactly("https://nxs.infy.tv/pbs/openrtb");
+    }
```

Comment:
What about to delete this test and add this assertion as part of `makeHttpRequestsShouldReturnHttpRequestWithCorrectBodyHeadersAndMethod` test?

Then it need to be renamed to `makeHttpRequestsShouldReturnExpectedHttpRequest ` or something similar.